packages/i18n/data/core.json

```diff
@@ -59,6 +59,7 @@
   "NavBar.Categories.list": "Categories list",
   "NavBar.Categories.search": "Categories search",
   "NavBar.Categories.title": "Categories",
+  "NavBar.CustomApplications.title": "Custom Applications",
```

Comment:
Do you wanna add it to Quick Access as well (behind a feature-toggle)? Just in case it didn't cross your mind.

Reply:
Ah right, good idea